.bazelrc

```
@@ -0,0 +1 @@
+import %workspace%/tools/preset.bazelrc
```

```suggestion
import %workspace%/tools/preset.bazelrc

--lockfile_mode=error
```

To make sure the lockfile is never out of date